create-proxy.sh

```diff
@@ -25,6 +25,30 @@ KEY_FILE=zabbix/enc/zabbix_proxy_key.pem
 CA_FILE=zabbix/enc/zabbix_proxy.ca
 START_CMD="docker-entrypoint.sh"
 
+read -p "Use default user 100 for UID (Y/n)?" -n 1 -r
+echo
+if [[ ${REPLY} =~ ^[nN]$ ]]; then
+  read -p 'Enter UID: ' user
+else
+  user=100
+fi
+
+read -p "Use default group 1000 for GID (Y/n)?" -n 1 -r
+echo
+if [[ ${REPLY} =~ ^[nN]$ ]]; then
+  read -p 'Enter GID: ' group
+else
+  group=1000
+fi
+
+read -p "Use default user 1000 for UID (Y/n)?" -n 1 -r
```

Comment:
This is duplicate? Same code twice?

Reply:
Looks like it is. Not sure which default UID it should be (100 or 1000). Usually first normal user is 1000 and user's GID matches UID.

Reply:
Looking code below updating the permissions the default in the proxy container is 100.

Reply:
Yeah it's duplicate. I'll remove it. We checked that zabbix users default UID and GID is 100:1000 so changed default values to match that.